README.md

```diff
@@ -193,8 +193,7 @@ The default (if the attribute is omitted or set to an invalid value) is ???.
     - TODO([#13](https://github.com/jackbsteinberg/std-toast/issues/13)): should this positioning be an attribute or a style
 - `closebutton`: a boolean attribute, determining whether an explicit close button is shown. 
```

Comment:
Is there a reason this is a boolean attribute? The `closebutton` attribute could be used, like the `showToast` option, to set the content of the closebutton. This would mean using this:

```html
<std-toast closebutton="Dismiss">
  Hello World!
</std-toast>
``` 

instead of

```html
<std-toast closebutton>
  Hello World!
  <span slot="closebutton">Dismiss</span>
</std-toast> 
```

Additionally, the `this.closebutton` reflector could be used to set the closebutton text from JS, which can only be done in the current state by adding a new HTML element to the toast.

Reply:
That's a great point. This is way better. I'll revise it.